<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallDialog</class>
 <widget class="QDialog" name="InstallDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ryzom Installer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainlLayout" stretch="1,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Welcome to Ryzom Installer!

This program will allow you to download, install, configure or manage Ryzom on your computer.

Just follow the different steps and make your choice between the different propositions.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="advancedLayout">
     <item>
      <spacer name="advancedHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="advancedCheckBox">
       <property name="text">
        <string>Show advanced parameters (expert)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="advancedFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="advancedMainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="sourceGroupBox">
        <property name="title">
         <string>Files will be installed from:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="oldDirectoryRadioButton">
           <property name="text">
            <string>Old installation: %1</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="anotherLocationLayout" stretch="1,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QRadioButton" name="anotherLocationRadioButton">
             <property name="text">
              <string>Another location: %1</string>
             </property>
             <property name="autoExclusive">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="anotherLocationBrowseButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QRadioButton" name="internetRadioButton">
           <property name="text">
            <string>Internet (%1 GiB to download)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="destinationGroupBox">
        <property name="title">
         <string>Files will be installed to (requires 10 GiB):</string>
        </property>
        <layout class="QHBoxLayout" name="destinationLayout" stretch="1,0">
         <item>
          <widget class="QLabel" name="destinationLabel">
           <property name="text">
            <string>c:\</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="destinationBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="clientArchGroupBox">
        <property name="title">
         <string>Do you prefer to use a 64 or 32 bits client?</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="clientArch64RadioButton">
           <property name="text">
            <string>64 bits (recommended)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="clientArch32RadioButton">
           <property name="text">
            <string>32 bits</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oldDirectoryRadioButton</tabstop>
  <tabstop>anotherLocationRadioButton</tabstop>
  <tabstop>anotherLocationBrowseButton</tabstop>
  <tabstop>internetRadioButton</tabstop>
  <tabstop>destinationBrowseButton</tabstop>
  <tabstop>clientArch64RadioButton</tabstop>
  <tabstop>clientArch32RadioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InstallDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InstallDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
